From cc0fb7c3d23779f17e2cf1da069c80abf614396e Mon Sep 17 00:00:00 2001
From: Danny Kukawka <danny.kukawka@web.de>
Date: Fri, 14 Aug 2009 16:05:58 +0200
Subject: [PATCH 06/33] s/playlist_formats/playlist_format/

Fix invalid keyname as pointed out in fd.o#23251:
- s/playlist_formats/playlist_format/
---
 .../10freedesktop/10-usb-music-players.fdi         |    8 ++++----
 1 files changed, 4 insertions(+), 4 deletions(-)

diff --git a/fdi/information/10freedesktop/10-usb-music-players.fdi b/fdi/information/10freedesktop/10-usb-music-players.fdi
index eaeb9f9..b92192f 100644
--- a/fdi/information/10freedesktop/10-usb-music-players.fdi
+++ b/fdi/information/10freedesktop/10-usb-music-players.fdi
@@ -232,7 +232,7 @@
             <append key="portable_audio_player.input_formats" type="strlist">audio/mpeg</append>
             <append key="portable_audio_player.audio_folders" type="strlist">MUSIC/</append>
             <append key="portable_audio_player.audio_folders" type="strlist">RECORD/</append>
-            <append key="portable_audio_player.playlist_formats" type="strlist">audio/x-mpegurl</append>
+            <append key="portable_audio_player.playlist_format" type="strlist">audio/x-mpegurl</append>
             <append key="portable_audio_player.playlist_path" type="string">PLAYLISTS/%File</append>
           </match>
         </match>
@@ -324,7 +324,7 @@
 	    <append key="portable_audio_player.input_formats" type="strlist">audio/x-wav</append>
 	    <append key="portable_audio_player.audio_folders" type="strlist">MUSIC</append>
 	    <append key="portable_audio_player.audio_folders" type="strlist">RECORD</append>
-	    <append key="portable_audio_player.playlist_formats" type="strlist">audio/x-mpegurl</append>
+	    <append key="portable_audio_player.playlist_format" type="strlist">audio/x-mpegurl</append>
 	    <append key="portable_audio_player.playlist_path" type="string">PLAYLISTS/%File</append>
 	  </match>
 	  <!-- Samsung YP-S5 -->
@@ -427,7 +427,7 @@
 	    <append key="portable_audio_player.output_formats" type="strlist">audio/x-ms-wma</append>
 	    <append key="portable_audio_player.input_formats" type="strlist">audio/x-wav</append>
 	    <append key="portable_audio_player.audio_folders" type="strlist">/</append>
-	    <append key="portable_audio_player.playlist_formats" type="strlist">audio/x-mpegurl</append>
+	    <append key="portable_audio_player.playlist_format" type="strlist">audio/x-mpegurl</append>
 	    <append key="portable_audio_player.playlist_path" type="string">PLAYLIST/%File</append>
 	    <match key="storage.lun" int="1">
 	      <merge key="storage.drive_type" type="string">sd_mmc</merge>
@@ -1104,7 +1104,7 @@
             <append key="portable_audio_player.audio_folders" type="strlist">music/</append>
             <append key="portable_audio_player.audio_folders" type="strlist">video/</append>
             <append key="portable_audio_player.audio_folders" type="strlist">record/</append>
-            <append key="portable_audio_player.playlist_formats" type="strlist">audio/x-mpegurl</append>
+            <append key="portable_audio_player.playlist_format" type="strlist">audio/x-mpegurl</append>
             <append key="portable_audio_player.playlist_path" type="string">playlists/%File</append>
           </match>
         </match>
-- 
1.7.1

